db/relational/create.sql

```diff
@@ -0,0 +1,311 @@
+-- phpMyAdmin SQL Dump
+-- version 4.8.3
+-- https://www.phpmyadmin.net/
+--
+-- Host: 127.0.0.1
+-- Creato il: Mar 16, 2020 alle 13:09
+-- Versione del server: 10.1.37-MariaDB
+-- Versione PHP: 7.2.12
+SET
+  SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+
+SET
+  AUTOCOMMIT = 0;
+
+START TRANSACTION;
+
+SET
+  time_zone = "+00:00";
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */
+;
+
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */
+;
+
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */
+;
+
+/*!40101 SET NAMES utf8mb4 */
+;
+
+--
+-- Database: `stalker-db`
+--
+-- --------------------------------------------------------
+--
+-- Struttura della tabella `admintype`
+--
+CREATE TABLE `admintype` (
+  `id` int(11) NOT NULL,
```

Comment:
valutare l'utilizzo di autoincrement

Reply:
è messo, in fondo al file ci sono le specifiche per l'autoincrement